cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -142,7 +144,8 @@ bool CobTwistController::initialize()
     {
         twist_controller_params_.frame_names.push_back(chain_.getSegment(i).getName());
     }
-    register_link_client_ = nh_.serviceClient<cob_srvs::SetString>("obstacle_distance/registerLinkOfInterest");
+
+    register_link_client_ = nh_.serviceClient<cob_srvs::SetString>("/register_links");
```

Comment:
This switch has not been finalzed/merged yet:https://github.com/ipa320/cob_robots/pull/467
Also there have been changes in uploading the SRDF that might have broken my preliminary workk on switching to moveit...don't know what the status is now...